python/requirements.txt

```diff
@@ -3,7 +3,7 @@ opentracing>=2,<3
 # We want something like: jaeger-client>=3.8,<4
 # But Scope Manager support not officially released in Jaeger at the time of writing.
 # This is using branch 'feature/opentracing-2.0-support' in @yurishkuro's fork.
--e git+https://github.com/yurishkuro/jaeger-client-python.git@d631596415cb549a4e8314cc25d6c53fab710528#egg=jaeger-client
+-e git+https://github.com/jaegertracing/jaeger-client-python.git@ef62773da0c273070190f736f921367c81c0e603#egg=jaeger-client
```

Comment:
Thanks for looking to fix this.  I think what's needed here is simply `jaeger-client>=4.3,<5`, and all the comments above can be removede

Reply:
Alright, let me read more on requirements.txt and I will update the pull request.